go.mod

```diff
@@ -263,6 +263,10 @@ replace (
 
 	go.universe.tf/metallb => github.com/cilium/metallb v0.1.1-0.20220829170633-5d7dfb1129f7
 
+	// Using private fork of client-go.
```

Comment:
nit; I see that the other pins also use the term `private`, but I think it would make more sense to just say `our fork` - given the fork is public and open source 😄

Reply:
thanks, done!